stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -13,42 +13,47 @@
 import com.stripe.exception.AuthenticationException;
 import com.stripe.net.RequestOptions;
 
+/**
+ * Class that handles {@link Token} creation from charges and {@link Card} models.
+ */
 public class Stripe {
-    private String defaultPublishableKey;
 
-    public TokenCreator tokenCreator = new TokenCreator() {
+    TokenCreator tokenCreator = new TokenCreator() {
         @Override
         public void create(
                 final Card card,
                 final String publishableKey,
                 final Executor executor,
                 final TokenCallback callback) {
-            AsyncTask<Void, Void, ResponseWrapper> task = new AsyncTask<Void, Void, ResponseWrapper>() {
-                protected ResponseWrapper doInBackground(Void... params) {
-                    try {
-                        RequestOptions requestOptions = RequestOptions.builder()
-                                .setApiKey(publishableKey).build();
-                        com.stripe.model.Token stripeToken = com.stripe.model.Token.create(
-                                hashMapFromCard(card), requestOptions);
-                        com.stripe.model.Card stripeCard = stripeToken.getCard();
-                        Card card = androidCardFromStripeCard(stripeCard);
-                        Token token = androidTokenFromStripeToken(card, stripeToken);
-                        return new ResponseWrapper(token, null);
-                    } catch (Exception e) {
-                        return new ResponseWrapper(null, e);
-                    }
-                }
-
-                protected void onPostExecute(ResponseWrapper result) {
-                    tokenTaskPostExecution(result, callback);
-               }
+            AsyncTask<Void, Void, ResponseWrapper> task =
```

Comment:
No logic changes here - just formatting and changing the method from public to package private. It was only public before because our tests were in the wrong package.